docs/guides/notebooks/configuration/groups.rst

```diff
@@ -0,0 +1,119 @@
+###################
+User-managed groups
+###################
+
+Create and manage closed user groups in order to share private files.
+
+.. _user-group-create:
+
+Create a group
+==============
+
+Groups are created an managed in the Comanage system at `id.lsst.cloud <https://id.lsst.cloud>`_
+(not from the terminal command line, as in some other systems).
+
+From the terminal command line it is possible to list all groups with ``getent group``.
```

Comment:
Does that actually work? For example it does not show me g_satsim which is a group that exists (but that I am not a member of).